component/org.wso2.carbon.identity.api.otp.service.emailotp/src/main/resources/email-otp.yaml

```diff
@@ -66,6 +66,45 @@ paths:
                 $ref: '#/components/schemas/Error'
       tags:
         - emailotp
+  /emailotp/verify:
+    post:
+      description: |
+        This API is used to verify the emailotp without invalidate.
+        <b>Permission required:</b><br>
+        * none<br>
+        <b>Scope required:</b><br>
+        * internal_login
```

Comment:
Is this correct?

Reply:
how is this API invoked?

Reply:
what is the permission that admin will need to invoke the api?

Reply:
Accounting to the requirement, this shout be the same as   /emailotp/validate API

Reply:
A separate git issue [1] was created as this mentioned issue should be addressed elsewhere as well
https://github.com/wso2/product-is/issues/14491